akka-stream-testkit/build.sbt

```diff
@@ -1,8 +1,6 @@
 import akka._
-import com.typesafe.tools.mima.plugin.MimaKeys
 
 AkkaBuild.defaultSettings
 Formatting.formatSettings
 OSGi.streamTestkit
 Dependencies.streamTestkit
```

Comment:
I mean we specifically call it out in releases: `Binary compatibility is not maintained for the following: ... akka-testkit
 ...`: http://akka.io/news/2016/02/17/akka-2.4.2-released.html 
It's a bit of a pain to fix mima if we know if the proj now BC bound hm